dla_cnn/desi/preprocess.py

```diff
@@ -92,3 +92,33 @@ def label_sightline(sightline, kernel, REST_RANGE, pos_sample_kernel_percent=0.3
     # classification is 1 / 0 / -1 for DLA/nonDLA/border
     # offsets_array is offset
     return classification, offsets_array, column_density
+
+def _normalize(sightline, camera):
+    '''
+    Normalize spectrum by dividing the mean value of continnum at lambda[left,right]
+    ------------------------------------------
+    parameters:
+    
+    sightline: dla_cnn.data_model.Sightline.Sightline object;
+    camera : str, 'b' : the blue channel of the specctra, 'r': the r channel of the spectra,
+                  'z' : the z channel of the spectra, 'all': all spectra
+    
+    --------------------------------------------
+    return
+    
+    sightline: the sightline after normalized
+    
+    '''
+    def _normalize(sightline,left,right):
```

Comment:
please rename this method to not be identical to the parent method.
too confusing.

Reply:
And add a short doc string for it too.